### PR TITLE
Log API update

### PR DESCRIPTION
**Pull request by mihaidma**

### CODE REVIEW

File: src/pages/api/index.md [hunk 1]
```diff
@@ -420,17 +420,46 @@ The name of the object or function you wish to export.
 ## log.level([entry, ..])
 The `log.x` method set provides the ability to log.
 
+- Supported log levels are: `debug`, `info`, `warn`, `error`, `fatal`
```
Comment (mcdonnelldean) [inline src/pages/api/index.md]: These are notes, add them below usage section

File: src/pages/api/index.md [hunk 2]
```diff
@@ -420,17 +420,46 @@ The name of the object or function you wish to export.
...
+- Logging suports level+ syntax: info+ means info and above: info, warn, error, fatal
+
+Convenience shortcut log levels:
```
Comment (mcdonnelldean) [inline src/pages/api/index.md]: Also notes

File: src/pages/api/index.md [hunk 3]
```diff
@@ -420,17 +420,46 @@ The name of the object or function you wish to export.
 ## log.level([entry, ..])
 The `log.x` method set provides the ability to log.
```
Comment (mcdonnelldean) [inline src/pages/api/index.md]: The `log.level` method set provides the ability to log directly off the seneca instance. By default we include a logging adaptor that emits json based logs. You can swap out this adaptor for others; see [Logging with Seneca][] for more information on adaptors

File: src/pages/api/index.md [hunk 4]
```diff
@@ -420,17 +420,46 @@ The name of the object or function you wish to export.
...
+seneca.log.error('error log level')
+seneca.log.fatal('fatal log level')
+
+// will output:
+// ["warn log level"]
+// ["error log level"]
+// ["fatal log level"]
+
+```
+
```
Comment (mcdonnelldean) [inline src/pages/api/index.md]: add same example above but set each message as {notice: 'whatever the message is'} to show json output

**Comment by mihaidma** [comment]
@mcdonnelldean updated

**Comment by mihaidma** [comment]
@mcdonnelldean i'll merge as all requested changes are done

**Pull request merged**